### PR TITLE
Branch bugfixes v3.0.1

### PR DESCRIPTION
**Pull request by deinhofer**
Version 3.0.1: Bug fix release

    Download deployment model: wrong attribute modelVersion: https://github.com/asterics/AsTeRICS/issues/240
    Improved tracking quality of XFaceTrackerLK plugin, don't reinit tracker that often: https://github.com/asterics/AsTeRICS/issues/238
    Fixed ACS in case it is started with a file as parameter: https://github.com/asterics/AsTeRICS/issues/232
    ARE is not dependent on JRE 8 but JRE7 was embedded in installer: https://github.com/asterics/AsTeRICS/issues/241

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -223,4 +223,10 @@ Release with major changes under the hood regarding ARE licensing and customized
 * Many plugin improvements.
 * Made REST API ports configurable in areProperties.
 * For further changes see the (Milestone 3.0)[https://github.com/asterics/AsTeRICS/milestone/5] and issues
+
+### Version 3.0.1: Bug fix release
+* Download deployment model: wrong attribute modelVersion: https://github.com/asterics/AsTeRICS/issues/240
+* Improved tracking quality of XFaceTrackerLK plugin, don't reinit tracker that often: https://github.com/asterics/AsTeRICS/issues/238
+* Fixed ACS in case it is started with a file as parameter: https://github.com/asterics/AsTeRICS/issues/232
```
Comment (klues) [inline RELEASE_NOTES.md]: can't find this in the current pull request. did you commit it directly to master?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -223,4 +223,10 @@ Release with major changes under the hood regarding ARE licensing and customized
...
+* ARE is not dependent on JRE 8 but JRE7 was embedded in installer: https://github.com/asterics/AsTeRICS/issues/241
```
Comment (klues) [inline RELEASE_NOTES.md]: also can't find this in the current pull request. isn't there a property that points to the jre8 path now, like normally in APE properties?

File: RELEASE_NOTES.md (hunk 1 above)
Comment (deinhofer) [reply RELEASE_NOTES.md]: I just added the workaround: the file ACS.bat (see at the bottom of this page)

File: RELEASE_NOTES.md (hunk 2 above)
Comment (deinhofer) [reply RELEASE_NOTES.md]: No, this is not visible within git, as it is not based on APE but built with the old approach which requires to simply provide a java folder at a certain location, which I changed locally to JRE 8.

**Pull request merged**